tests/security/selinux/selinux_setup.pm

```diff
@@ -17,59 +18,66 @@ sub run {
+        'libsemanage1', 'libsemanage-devel', 'checkpolicy', 'mcstrans', 'restorecond', 'setools-console',
+        'setools-devel', 'setools-java', 'setools-libs', 'setools-tcl'
     );
     foreach my $pkg (@pkgs) {
         my $results = script_run("zypper --non-interactive se $pkg");
         if ($results) {
-            record_info("WARNING", "Package $pkg is missing, zypper search returns: $results");
+            record_info('WARNING', "Package $pkg is missing, zypper search returns: $results");
         }
         else {
             zypper_call("in $pkg");
         }
     }
     if (is_x86_64) {
-        zypper_call("in libselinux1-32bit");
+        zypper_call('in libselinux1-32bit');
     }
 
-    # for opensuse, e.g, Tumbleweed install selinux_policy pkgs as needed
-    # for sle15 and sle15+ "selinux-policy-*" pkgs will not be released
+    # For opensuse, e.g, Tumbleweed install selinux_policy pkgs as needed
+    # For sle15 and sle15+ "selinux-policy-*" pkgs will not be released
     # NOTE: have to install "selinux-policy-minimum-*" pkg due to this bug: bsc#1108949
-    # install policy packages separately due to this bug: bsc#1177675
+    # Install policy packages separately due to this bug: bsc#1177675
     if (is_sle('>=15')) {
         my @files
-          = ("selinux-policy-20200219-3.6.noarch.rpm", "selinux-policy-minimum-20200219-3.6.noarch.rpm", "selinux-policy-devel-20200219-3.20.noarch.rpm");
+          = ('selinux-policy-20200219-3.6.noarch.rpm', 'selinux-policy-minimum-20200219-3.6.noarch.rpm', 'selinux-policy-devel-20200219-3.20.noarch.rpm');
         foreach my $file (@files) {
-            assert_script_run "wget --quiet " . data_url("selinux/$file");
+            assert_script_run 'wget --quiet ' . data_url("selinux/$file");
             assert_script_run("rpm -ivh --nosignature --nodeps --noplugins $file");
         }
     }
     elsif (!is_sle && !is_leap) {
-        my @files = ("selinux-policy", "selinux-policy-minimum", "selinux-policy-devel");
+        my @files = ('selinux-policy', 'selinux-policy-minimum', 'selinux-policy-devel');
         foreach my $file (@files) {
             zypper_call("in $file");
         }
     } else {
-        zypper_call("in selinux-policy-minimum");
+        zypper_call('in selinux-policy-minimum');
     }
 
-    # record the pkgs' version for reference
-    my $results = script_output("zypper se -s selinux-policy");
-    record_info("Pkg_ver", "SELinux policy packages' version is: $results");
+    # Record the pkgs' version for reference
+    my $results = script_output('zypper se -s selinux policy');
+    record_info('Pkg_ver', "SELinux packages' version is: $results");
+
+    # Check SELinux status by default, it should be disabled
+    # 'selinuxenabled' exits with status 0 if SELinux is enabled and 1 if it is not enabled
+    validate_script_output('sestatus', sub { m/SELinux status: .*disabled/ });
+    if (script_run('selinuxenabled') == 0) {
```

Comment:
The result of 'sestatus' is not enough?  if check the 'selinuxenabled' is necessary?

Reply:
It is enough actually but I added "selinuxenabled" here for testing "selinuxenabled" itself. This testing is really simple so I added here.